### ISSUE TITLE
iOS Build issue with `analytics-react-native-firebase`

### ISSUE THREAD

**Issue by kbrandwijk** [segmentio/analytics-react-native#154]
The latest version (1.1.1-beta.0) causes a build issue on the latest version of RN (0.62).

`fatal error: 'Segment_Firebase/SEGFirebaseIntegrationFactory.h' file not found`

The reason is this line in `main.m`: 
```
#import <Segment_Firebase/SEGFirebaseIntegrationFactory.h>
```

Apparently, there used to be some naming convention issue that caused `-` to become `_`, but that's no longer the case. Changing it back to:

```
#import <Segment-Firebase/SEGFirebaseIntegrationFactory.h>
```

solved the build issue.

**Comment by bsneed** [comment]
We'll release a new beta later this week to correct this.

**Comment by bsneed** [comment]
Fixes pending in PR #157
